src/components/pages/Home.tsx

```diff
@@ -47,36 +52,75 @@ function LoaderArea() {
 function SectionQueryResult({
   title,
   query,
+  queryType,
   action
 }: {
   title: ReactElement | string
-  query: SearchQuery
+  query?: SearchQuery
+  queryType?: string
   action?: ReactElement
 }) {
   const { config } = useOcean()
   const [result, setResult] = useState<QueryResult>()
   const [loading, setLoading] = useState(true)
+  const { data } = useQuery(getHighestLiquidityAssets)
 
   useEffect(() => {
-    if (!config?.metadataCacheUri) return
-
+    if (!config?.metadataCacheUri || !data) return
     const source = axios.CancelToken.source()
 
     async function init() {
-      const result = await queryMetadata(
-        query,
-        config.metadataCacheUri,
-        source.token
-      )
-      setResult(result)
-      setLoading(false)
+      if (queryType && queryType === 'graph') {
+        const ddoList: DDO[] = []
+        setLoading(true)
+        try {
```

Comment:
this can be made into one `try/catch` where, if all the if/else can't be refactored, the if/else clauses happen within the `try` block

Then in all errors, we should log like so instead of plain `console.log`:

```js
Logger.error(error.message)
```